frontend/server/src/DAO/UserRank.php

```diff
@@ -164,4 +164,66 @@ public static function getAuthorsRank(
             'total' => $totalRows,
         ];
     }
+
+    /**
+     * @return array{ranking: list<array{author_ranking: int, name: null|string, username: string}>, total: int}
+     */
+    public static function getAuthorsRankWithQualityProblems(
+        int $page,
+        int $rowsPerPage
+    ): array {
+        $sqlSelect = '
+            SELECT
+                `ur`.`author_ranking`,
```

Comment:
```suggestion
                IFNULL(`ur`.`author_ranking`, 0),
```

con eso debería bastar para que se deje de quejar las pruebas.